test/models/ui/UIComponentsSpec.scala

```diff
@@ -125,142 +135,114 @@ class UIComponentsSpec extends BaseSpec {
+      val bp1LinkCy: Link = Link("http://bulletPointOneUrl","Welsh bullet point 1 link", false)
 
-      val bulletPointListLeadingText: Seq[TextItem] =
-        Seq( leadingTextFirstTextComponent, leadingTextHyperLink, leadingTextSecondTextComponent )
+      val bp2TxtEn: Words = Words( "Bullet point 2 text" )
+      val bp2TxtCy: Words = Words( "Welsh bullet point 2 text" )
 
+      val bp2LinkEn: Link = Link("http://bulletPointTwoUrl", "Bullet point 2 link", true)
+      val bp2LinkCy: Link = Link("http://bulletPointTwoUrl", "Welsh bullet point 2 link", true)
 
-      val bulletPointListItems: Seq[Seq[TextItem]] = Seq(
-        Seq( bulletPointOneText, bulletPointOneHyperLink ),
-        Seq( bulletPointTwoText, bulletPointTwoHyperLink )
+      val bpListLeadingText: Text =
+        Text(Seq( leadingTxt1En, leadingLinkEn, leadingTxt2En ),
+             Seq( leadingTxt1Cy, leadingLinkCy, leadingTxt2Cy ) )
+
+
+      val bulletPointListItems: Seq[Text] = Seq(
+        Text(Seq(bp1TxtEn, bp1LinkEn),Seq(bp1TxtCy, bp1LinkCy)),
+        Text(Seq(bp2TxtEn, bp2LinkEn),Seq(bp2TxtCy, bp2LinkCy))
       )
 
-      val bulletPointList: BulletPointList = BulletPointList( bulletPointListLeadingText, bulletPointListItems )
+      val bulletPointList: BulletPointList = BulletPointList( bpListLeadingText, bulletPointListItems )
 
       // Test components of bullet point list
-      bulletPointList.leadingText.head.toString mustBe "[Leading text 1:Welsh leading text 1:false]"
-      bulletPointList.leadingText(1).toString mustBe "[link:[Leading text link:Welsh leading text link:false]:http://leadingTextUrl:false]"
-      bulletPointList.leadingText(2).toString mustBe "[Leading text 2:Welsh leading text 2:false]"
+      bulletPointList.leadingText.english(0).toString mustBe "Leading text 1"
+      bulletPointList.leadingText.english(1).toString mustBe "[link:Leading text link:http://leadingTextUrl:false]"
+      bulletPointList.leadingText.english(2).toString mustBe "Leading text 2"
 
-      bulletPointList.listItems.head.head.toString mustBe "[Bullet point 1 text:Welsh bullet point 1 text:false]"
-      bulletPointList.listItems.head(1).toString mustBe "[link:[Bullet point 1 link:Welsh bullet point 1 link:false]:http://bulletPointOneUrl:false]"
+      bulletPointList.listItems.head.english(0).toString mustBe "Bullet point 1 text"
+      bulletPointList.listItems.head.english(1).toString mustBe "[link:Bullet point 1 link:http://bulletPointOneUrl:false]"
 
-      bulletPointList.listItems(1).head.toString mustBe "[Bullet point 2 text:Welsh bullet point 2 text:false]"
-      bulletPointList.listItems(1)(1).toString mustBe "[link:[Bullet point 2 link:Welsh bullet point 2 link:false]:http://bulletPointTwoUrl:true]"
+      bulletPointList.listItems(1).english(0).toString mustBe "Bullet point 2 text"
+      bulletPointList.listItems(1).english(1).toString mustBe "[link:Bullet point 2 link:http://bulletPointTwoUrl:true]"
```

Comment:
This is probably a pre-existing problem, but at present we check only the english version of the bullet point list. For completeness I suggest we check the Welsh as well.

Reply:
Yes, this reflects the original code

Reply:
I suggest we fix this when we fix the bullet point list bug!